docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -17,6 +17,23 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Distributed Tracing in Grafana Cloud k6
+
+<!-- Heitor Tashiro Sergent -->
+
+_Generally available in Grafana Cloud_
+
+You can now use the Grafana Cloud Traces integration with Grafana Cloud k6 to quickly debug failed performance tests and proactively improve application reliability.
+
+Distributed tracing in Grafana Cloud k6 only requires two things:
+
+- An application instrumented for tracing with Grafana Cloud Traces.
+- Adding a few lines of code to your existing k6 scripts.
+
+The integration works by having k6 inject tracing metadata into the requests it sends to users’ backend services when they run a test. The tracing data is then correlated with k6 test run data, so users can understand how their services and operations behaved during the whole test run. The collected tracing data is aggregated to generate real-time metrics, such as frequency of calls, error rates, and percentile latencies, that help users narrow their search space and quickly spot anomalies.
```

Comment:
```suggestion
The integration works by having k6 inject tracing metadata into the requests it sends to users’ backend services when they run a test. The tracing data is then correlated with k6 test run data, so users can understand how their services and operations behaved during the whole test run. The collected tracing data is aggregated to generate real-time metrics—such as frequency of calls, error rates, and percentile latencies—that help users narrow their search space and quickly spot anomalies.
```

Reply:
Wondering if we can change "users" to "you" in the paragraph above or are we talking about "your users"?